python/tests/toolchains/defs.bzl

```diff
@@ -17,6 +17,12 @@
 
 load("//python:versions.bzl", "PLATFORMS", "TOOL_VERSIONS")
 
+_WINDOWS_RUNNER_TEMPLATE = """\
+@ECHO OFF
+set PATHEXT=.COM;.EXE;.BAT
```

Comment:
The way windows figures out something is executable is through this environment variable.